test/ELMDependencyHelper.test.ts

```diff
@@ -130,6 +130,9 @@ describe('ELMDependencyHelper', () => {
         accessLevel: 'Public'
       };
       const vs = ELMDependencyHelper.findValueSetReference(elm, [elm, dependency], valueSetRef);
+      if (vs) {
+        vs.annotation = undefined;
+      }
```

Comment:
Is the purpose here just to get rid of the annotation property on the valueset? If so, `delete vs.annotation` is a way of doing that that makes the goal a little clearer